### DISCUSSION TITLE
時系列SNS or 旅のしおりアプリ なのかはっきりさせたい

### DISCUSSION THREAD

**Discussion by Takapy1** [Ayato-kosaka/react-itinerary#666] [General]
@Ayato-kosaka 

あまりまとまってなくて、申し訳ない！
ただ、俺的には事業価値固まってない状態で、続けるのがしっくりこないから先に方針決めたいと思った。
まだ時系列SNSのイメージができてないから、そこを詳しくヒアリングしたい。

こさの思考として
資金調達の時に事業価値のインパクトが足りないから、時系列SNSに切り替えた方がいいと思っている場合
→意見：まだニーズが明確じゃないから、時系列SNS事業のポテンシャル検討後に判断すべき

そもそも旅のしおりアプリはやりたいことと違う場合
→意見：絶対に今のうちに時系列SNSに切り替えた方がいい、

だと思う。
これは明日話し合いたい.

# 議題
時系列SNS or 旅のしおりアプリ なのかはっきりさせたい

## 理由
2つの選択肢で、事業価値(どの顧客の、どの課題）の部分が変わるので、
→解決方法も変わる
→市場、顧客ターゲットも変わるので、
→提供方法も変わる
→マーケティング戦略も変わる
→ビジネスモデル(集客の仕組みなど）が変わる

⚠️アプリなので収益モデルは変わらない
２つの共通で資料作りできるのが収益モデルと企業価値だけになってしまう

### 顧客課題の再策定
- なぜこれをやるのか (自分の仮説）
- 顧客ニーズの調査(市場に顕在問題）
- アプリのいくつかの部分変更(後でいい）

**Comment by Ayato-kosaka** [comment]
@Takapy1 
確かにねぇー、めちゃくちゃ申し訳ない！

そこ固まらないと他進めないのは承知した！

→取り急ぎ初版作りたいから、リリースを来週の月曜に見送って先企業価値のスライド作るでいい？
→その時に野口の手が空きそうだったらサムネイルの残タスク少しお願いしたい！（マーケティングでやることあるなら無問題）


> 　資金調達の時に事業価値のインパクトが足りないから、時系列SNSに切り替えた方がいいと思っている場合
そもそも旅のしおりアプリはやりたいことと違う場合

これはどっちもなんだよね。
作ってるアプリに間違いないから、やりたいことと違うわけではないけど、ちょっと幅が広がるイメージ（前回話した、家で過ごすとかが含まれてくる）

**Comment by Ayato-kosaka** [comment]
@Takapy1 

取り急ぎ、事業価値の概要の部分書いてみた！
ただ、今回の場合、課題がうまく書けなかったから、曖昧な文章になってる！

```

世界を変える新しいSNSが登場します。それは、あなたの人生を時系列で投稿できるSNSです。Twitterは文字、Instagramは写真を投稿するSNSであったのに対し、私たちの特別な一日を時系列として投稿することができます。これによって、私たちの人生を最も価値あるものに差別化し、世界中の人々に共有することができます。

従来の投稿方法では、ある場所に行ったことを投稿するだけでした。しかし、時系列投稿によって、その場所での体験や出来事をストーリーとして表現することができます。これによって、私たち自身の人生をより豊かに表現することができます。

また、時系列投稿は自分自身を表現するための非常にパワフルなツールでもあります。私たちは自分自身の価値観や趣味、ライフスタイルを時系列として投稿し、他の人々と共有することができます。これによって、私たちの人生はさらに多様化し、世界中の人々が互いの人生を知り合い、理解し合うことができるようになります。

時系列投稿は、あなたの人生を差別化し、豊かに表現することができる非常にパワフルなツールです。私たちは、世界中の人々と共有することで、より豊かな人生を送ることができます。この新しいSNSによって、私たちは世界を変えることができます。


```

**Reply by Ayato-kosaka** [reply]
@Takapy1 

一旦思い立った経緯を含めて、メッセージ作ってみた！
野口のインプットとして読んで欲しいのと、
方向性として違和感ないか確認をお願いします！（まだ、方向性ぐらいだから細かいレビューは不要です！）

私は、旅行は、どこに行ったかよりも、その場所をどう周ったか、どんな冒険を経験したか、誰と一緒に旅をしたかが重要だと思います。

また、人生も同様で、単にどんな経験をしたかではなく、それらをどんな順序で経験したかが大事だと思います。

人生は、多くの挑戦や喜び、悲しみや失敗を経験することで成り立っています。人生は、それらの経験をただ羅列するのではなく、その順序やつながりを意識した物語です。

そこで私が提案するのは、時系列を投稿するSNSアプリSpelieveです。このアプリでは、自分の旅の物語や特別な一日、今年の楽しかった思い出などを細かく時系列に沿って投稿することができます。他のユーザーがコメントやいいねを残すことで、共感や感動を共有することができます。

例えば、結婚式という特別な一日を投稿するときも、Spelieveなら準備段階から始めることができます。ドレス選びや会場探し、招待状作りなどを時系列に投稿することで、自分自身や他のユーザーが一緒に過ごした特別な時間を振り返ることができます。

また、今年の振り返りとして、1月から12月までの出来事を時系列に沿って投稿することもできます。旅行やレストラン、イベントなど、楽しい思い出を投稿することで、他のユーザーと共感し合うことができます。

Spelieve なら、それらの特別な一日、特別な一年を投稿を通して素敵な思い出として情報を圧縮されることで、人生の旅を表現し、発信することができます。
